src/surfaceModels.ts

```diff
@@ -26,9 +26,9 @@ export class MeshColors {
 
 export class Surface {
   public mesh: SurfaceMesh;
-  public colors: MeshColors;
+  public colors: MeshColors | undefined;
 
-  constructor(mesh: SurfaceMesh, colors: MeshColors) {
+  constructor(mesh: SurfaceMesh, colors: MeshColors | undefined = undefined) {
```

Comment:
Instead of obj | undefined use the shorthand here and above

colors?: MeshColor

Reply:
Good catch! Done.